client/app/components/sections/Topbar/Topbar.css

```diff
@@ -54,16 +54,29 @@
     }
   }
 
+  & button {
```

Comment:
Since we've ended up using tag selector (I trust it was a well-considered choice), I'd like to make it as specific as possible. So maybe a child selector (`& > button`) instead of a more broader descendant selector?